### PR TITLE
Aesthetic fixes

### PR DESCRIPTION
**Pull request by chemikadze**

### CODE REVIEW

**Comment by chemikadze** [comment]
@dmakhno take a look

**Comment by chemikadze** [comment]
@dmakhno @rtimush up

File: meta.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ kit:
   applications:
     - name: "PetClinic on Docker"
       manifest: https://raw.github.com/qubell-bazaar/component-petclinic-docker/master/petclinic.yml
-    - name: "WAR on Docker"
+    - name: "Web application builder on Docker"
```
Comment (dornatsky) [inline meta.yml:7]: the root app manifest should be updated as well

Comment (dornatsky) [inline meta.yml:7]: disregard, I added this app from your fork, not realizing we have absolute paths in the metafile

**Pull request merged**